diff --git a/src/Plugin/views/exposed_form/BetterExposedFilters.php b/src/Plugin/views/exposed_form/BetterExposedFilters.php
index fbcb258..2745f1f 100644
--- a/src/Plugin/views/exposed_form/BetterExposedFilters.php
+++ b/src/Plugin/views/exposed_form/BetterExposedFilters.php
@@ -1160,6 +1160,7 @@ Title Desc|Z -> A</pre> Leave the replacement text blank to remove an option alt
             // Single-select -- display as radio buttons.
             $form[$field_id]['#type'] = 'radios';
             $form[$field_id]['#theme'] = 'bef_radios';
+            $form[$field_id]['#summary_attributes'] = [];
             if (empty($form[$field_id]['#process'])) {
               $form[$field_id]['#process'] = array();
             }
@@ -1168,6 +1169,7 @@ Title Desc|Z -> A</pre> Leave the replacement text blank to remove an option alt
           else {
             $form[$field_id]['#type'] = 'checkboxes';
             $form[$field_id]['#theme'] = 'bef_checkboxes';
+            $form[$field_id]['#summary_attributes'] = [];
 
             if ($options['more_options']['bef_select_all_none'] || $options['more_options']['bef_select_all_none_nested']) {
               $form[$field_id]['#attached']['library'][] = 'better_exposed_filters/select_all_none';
