diff --git a/js/eu_cookie_compliance.js b/js/eu_cookie_compliance.js
index bcb574f..0da4328 100644
--- a/js/eu_cookie_compliance.js
+++ b/js/eu_cookie_compliance.js
@@ -261,10 +261,10 @@
     Drupal.eu_cookie_compliance.setStatus(0);
     let popup = $('#sliding-popup');
     if (popup.hasClass('sliding-popup-top')) {
-      popup.animate({ top: popup.outerHeight() * -1 }).trigger('eu_cookie_compliance_popup_close');
+      popup.animate({ top: popup.outerHeight() * -1 }).trigger('eu_cookie_compliance_popup_close').remove();
     }
     else {
-      popup.animate({ bottom: popup.outerHeight() * -1 }).trigger('eu_cookie_compliance_popup_close');
+      popup.animate({ bottom: popup.outerHeight() * -1 }).trigger('eu_cookie_compliance_popup_close').remove();
     }
   };