diff --git src/Feeds/Processor/EntityProcessorBase.php src/Feeds/Processor/EntityProcessorBase.php
index baad3067..e15c0df0 100644
--- src/Feeds/Processor/EntityProcessorBase.php
+++ src/Feeds/Processor/EntityProcessorBase.php
@@ -227,7 +227,7 @@ abstract class EntityProcessorBase extends ProcessorBase implements EntityProces
     // Set list of entities to clean.
     $ids = $this->queryFactory->get($this->entityType())
       ->condition('feeds_item.target_id', $feed->id())
-      ->condition('feeds_item.hash', $this->getConfiguration('update_non_existent'), '<>')
+      // ->condition('feeds_item.hash', $this->getConfiguration('update_non_existent'), '<>')
       ->execute();
     $state->setList($ids);
